ginga/rv/plugins/Catalogs.py

```diff
@@ -372,26 +401,38 @@ def redo(self):
             return True
 
         # Copy the image parameters out to the widget
-        d = { 'ra': ra_ctr, 'dec': dec_ctr, 'width': str(wd),
-              'height': ht, 'r': radius, 'r2': radius,
-              'r1': 0.0,
-              }
+        d = {'ra': ra_ctr, 'dec': dec_ctr, 'width': str(wd),
+             'height': ht, 'r': radius, 'r2': radius,
+             'r1': 0.0}
         self._update_widgets(d)
         return True
 
-    def btndown(self, canvas, event, data_x, data_y):
+    def btndown(self, canvas, event, data_x, data_y, viewer):
         return True
 
-    def btnup(self, canvas, event, data_x, data_y):
-
-        objs = self.canvas.get_items_at(data_x, data_y)
+    def btnup(self, canvas, event, data_x, data_y, viewer):
```

Comment:
1. Open `s1_mips_v0.30_sci.fits` (I put it on Box for you).
2. Open `Catalog` with [this ginga_config.py](https://github.com/spacetelescope/stginga/blob/master/stginga/examples/configs/ginga_config.py).
3. Select `GSC_1.2` and search.
4. Left click on any of the circled stars on the image in an attempt to highlight the corresponding table entry.

I see this (without this PR):
```
2017-05-19 16:57:14,716 | E | Callback.py:139 (make_callback) | Error making callback 'cursor-up': reduce() of empty sequence with no initial value
2017-05-19 16:57:14,717 | E | Callback.py:140 (make_callback) | Traceback:
  File ".../ginga/misc/Callback.py", line 123, in make_callback
    res = method(*cb_args, **cb_kwdargs)

  File ".../ginga/rv/plugins/Catalogs.py", line 387, in btnup
    objs = self.canvas.get_items_at(data_x, data_y)

  File ".../ginga/canvas/CompoundMixin.py", line 67, in get_items_at
    if obj.contains(x, y):

  File ".../ginga/canvas/CompoundMixin.py", line 61, in contains
    res = self.contains_arr(x_arr, y_arr)

  File ".../ginga/canvas/CompoundMixin.py", line 57, in contains_arr
    self.objects))
```